tests/tracing/test_integration_tests.py

```diff
@@ -39,21 +36,23 @@ def test_basic(sentry_init, capture_events, sample_rate):
         span1, span2 = event["spans"]
         parent_span = event
         assert span1["tags"]["status"] == "internal_error"
+        assert span1["status"] == "internal_error"
         assert span1["op"] == "foo"
         assert span1["description"] == "foodesc"
         assert "status" not in span2.get("tags", {})
         assert span2["op"] == "bar"
         assert span2["description"] == "bardesc"
         assert parent_span["transaction"] == "hi"
-        assert "status" not in event["tags"]
+        assert "status" not in event.get("tags", {})
         assert event["contexts"]["trace"]["status"] == "ok"
     else:
         assert not events
 
 
-@pytest.mark.parametrize("sampled", [True, False, None])
 @pytest.mark.parametrize("sample_rate", [0.0, 1.0])
-def test_continue_from_headers(sentry_init, capture_envelopes, sampled, sample_rate):
+def test_continue_trace(
```

Comment:
simplified this test since it was too confusing and we can write a separate test for explicit sampling later if really wanted